documentation/cml_on_hyperv.md

```diff
@@ -0,0 +1,58 @@
+# CML on Hyper-V
+This installs CML on Hyper-V for Windows. You will need a version of windows that supports and provides Hyper-V as a feature. Editions other than Pro and Enterprise likely do not include this feature. Server 2016 and later should include Hyper-V, although the process below may change slightly if you are running this on a server instance.
+
+This was tested on Windows 11 Enterprise, but should work for other editions of windows that meet the requirements.
+
+## Requirements
+- Windows 10 Pro/Enterprise or later, Server 2016 or later
+- Hyper-V windows feature activated
+
+## Setup
+### Enable Hyper-V
+- Windows Settings: "Turn Windows features on or off"
+- Enable all Hyper-V options
+- Reboot when asked
+
+### Create Hyper-V Bridged Network
+This is done to expose the VM to your local network. If you only plan on accessing the CML instances from your local machine only, you can skip this step.
+- Launch the Hyper-V GUI
+- Click the "Virtual Switch Manager"
+- Create a new "External" virtual switch
+- Pick a network card connected to your local network
+- Confirm and save
+
+### Create Hyper-V Virtual Machine
+Before proceeding with the following steps, make sure to download the latest CML ISO for bare metal installation.
+
+#### Initial Creation
+- Create a new virtual machine
+- Use a name without spaces to avoid quoting/escaping issues
+- Select _Generation 2_ for the generation type
+- Assign sufficient memory to meet requirements
+- Do not use dynamic memory (this can impact performance on the nested nodes as page swapping may occur)
```

Comment:
You might want to separate that out as a tip or warning.  For example:

```markdown
- Do not use dynamic memory.
  - Attention: Using dynamic memory with the CML VM will have a negative impact on the performance on your labs running in the CML VM.  The nodes in your network simulation use nested virutalization, and using dynamic memory in Hyper-V can cause _significant_ performance problems because of page swapping.
```